spread.yaml

```diff
@@ -148,6 +148,17 @@ backends:
                 username: test
                 password: ubuntu
 
+    nested:
```

Comment:
Why do we need a new backend for this, given that we already have a Linode backend with a system matching it exactly. The single test below can be in its own suite, as it is now, and that specific test can declare support for `systems: [ubuntu-16.04-64]` only. It can also tune its `halt-timeout` and `kill-timeout` locally.

If we don't want to run it by default next to our other tests, we can just mark it as `manual: true`, or even mark the whole suite as `manual: true`.